app/updater.js

```diff
@@ -131,17 +131,16 @@ var requestVersionInfo = (done, pingOnly) => {
   if (!platformBaseUrl) throw new Error('platformBaseUrl not set')
 
   // Get the daily, week of year and month update checks
-  var lastCheckYMD = AppStore.getState().toJS().updates['lastCheckYMD'] || null
+  const state = AppStore.getState()
+  var lastCheckYMD = state.getIn(['updates', 'lastCheckYMD']) || null
```

Comment:
Merged it all the way up to 0.17.x but obviously no respin for that, just merged it up for the sake of easier merges if anything else changes.